gyro/lib/TaskManager.js

```diff
@@ -438,14 +438,22 @@ TaskManager.prototype._runNext = function() {
 TaskManager.prototype._runTask = function(taskName) {
     var task = this._tasks[taskName];
 
+
+    /* Some tasks spawn their own processes or heavily use
+        system resources other than CPU. Use config.parallelWeight to offset this */
+    var taskConfig = task.getConfig() || {};
+    var parallelWeight = parseInt(taskConfig['parallelWeight'] || 1);
```

Comment:
Just because it wasn't clear to me in the beginning, could you maybe put a comment here that says something about `parallelWeight` is the number of tasks per running process? (is that right?)